.github/lighthouse/lighthouserc.json

```
@@ -0,0 +1,112 @@
+        "doctype": ["error"],
+        "charset": ["error"],
+
+        "html-has-lang": ["error"],
+        "html-lang-valid": ["error"],
+        "document-title": ["error"],
+        "meta-viewport": ["error"],
+        "landmark-one-main": ["error"],
+        "link-name": ["warn"],
+        "link-in-text-block": ["warn"],
+        "frame-title": ["error"],
+        "image-alt": ["error"],
```

Biome (2.3.14)

[error] The key image-alt was already declared.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)